pkg/cluster/reconcile.go

```diff
@@ -220,18 +220,21 @@ func (c *Cluster) disasterRecovery(left etcdutil.MemberSet) error {
 			backupNow = true
 		}
 	}
+	exist := true
```

Comment:
there is something i am not sure i understand here
if in recovery mode, and there are still pods running try back if failed do ```c.logger.Errorln(err)``` (line 218) that is it? What if the pods are not in quorum

Reply:
The logic would be like after 218.

1. we have exist initialized to be true. 
2. remove remaining pods
3. c.startSeedMember(exist), based on exist. It will recover using the backup initiated before. 

SG?